src-plugins/mscExportVideo/mscExportVideo.cpp

```diff
@@ -347,6 +372,30 @@ void ExportVideo::handleWidgetDisplayAccordingToType(int index)
         d->qualityLabel->hide();
         break;
     }
+    case FFMPEG:
+    {
+        // Video
+        d->exportDialog->selectFile("video.avi");
+        d->frameRateSpinBox->show();
+        d->frameRateLabel->show();
+        d->subsamplingComboBox->hide();
+        d->subsamplingLabel->hide();
+        d->qualityComboBox->hide();
+        d->qualityLabel->hide();
+        break;
+    }
+    case MPEG2:
+    {
+        // Video
+        d->exportDialog->selectFile("video.avi");
+        d->frameRateSpinBox->hide();
+        d->frameRateLabel->hide();
+        d->subsamplingComboBox->hide();
+        d->subsamplingLabel->hide();
+        d->qualityComboBox->hide();
+        d->qualityLabel->hide();
+        break;
+    }
     }
```

Comment:
'{' Indentation, or I still don't understand our indentation practice for switches

Reply:
(he says he's ok)